Test/Unit/Model/Api/UpdateCartTest.php

```diff
@@ -0,0 +1,545 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2020 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Test\Unit\Model\Api;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\Webapi\Exception as WebApiException;
+use Magento\Framework\Webapi\Rest\Response;
+use Bolt\Boltpay\Api\UpdateCartInterface;
+use Bolt\Boltpay\Api\Data\CartDataInterface;
+use Bolt\Boltpay\Api\Data\UpdateCartResultInterface;
+use Bolt\Boltpay\Model\Api\UpdateCartCommon;
+use Bolt\Boltpay\Model\Api\UpdateCartContext;
+use Bolt\Boltpay\Model\ErrorResponse as BoltErrorResponse;
+use Bolt\Boltpay\Api\Data\CartDataInterfaceFactory;
+use Bolt\Boltpay\Api\Data\UpdateCartResultInterfaceFactory;
+use Bolt\Boltpay\Helper\Session as SessionHelper;
+use Bolt\Boltpay\Helper\Log as LogHelper;
+use Bolt\Boltpay\Model\Api\UpdateCart;
+use Bolt\Boltpay\Test\Unit\TestHelper;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+
+/**
+ * Class UpdateCartTest
+ * @package Bolt\Boltpay\Test\Unit\Model\Api
+ * @coversDefaultClass \Bolt\Boltpay\Model\Api\UpdateCart
+ */
+class UpdateCartTest extends TestCase
+{
+    const PARENT_QUOTE_ID = "1000";
+    const IMMUTABLE_QUOTE_ID = "1001";
+    const INCREMENT_ID = '100050001';
+    const DISPLAY_ID = self::INCREMENT_ID . ' / ' . self::IMMUTABLE_QUOTE_ID;
```

Comment:
@dumega Same as above comment, this will break now with the changes to display_id.